packages/rest/test/integration/http-handler.integration.ts

```diff
@@ -449,40 +449,28 @@ describe('HttpHandler', () => {
     }
   });
 
-  context('x-skip-body-parsing', () => {
+  context('x-parser extension', () => {
```

Comment:
As I commented before, is this behavior really specific to HttpHandler? IMO, this should be tested in unit or integration tests for RequestParser class.

Reply:
Moved to `body-parser.unit.ts`